lib/index.js

```diff
@@ -22,6 +22,9 @@ function register (server, options, next) {
         if (err) {
           return cb(err, null);
         }
+        if (rows.length === 0) { // i.e. no orgs -OR- filter --> wrong tagId or no orgs data
+          return cb(err, null);
```

Comment:
Yeah. I think it is good where it is, I am just wondering what we want to return in this case.